code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -73,4 +73,16 @@
 
 /turf/simulated/floor/carpet/burn_tile()
 	burnt = 1
-	update_icon()
\ No newline at end of file
+	update_icon()
+
+
+
+/turf/simulated/floor/fakespace
+	icon = 'icons/turf/space.dmi'
+	icon_state = "0"
+	floor_tile = /obj/item/stack/tile/fakespace
+	broken_states = list("damaged")
+
+/turf/simulated/floor/fakespace/New()
+	..()
+	icon_state = "[rand(0,25)]"
```

Comment:
Perhaps so traitors could put down a bunch of fake space tiles with a real
one in the middle, and have them line up. Although I suppose it would still
be immediately obvious based on which tiles are affected by light.

On 9 August 2015 at 13:27, Cheridan <notifications@github.com> wrote:

> In code/game/turfs/simulated/floor/fancy_floor.dm
> <https://github.com/tgstation/-tg-station/pull/11057#discussion_r36583451>
> :
>
> > \ No newline at end of file
> > +	update_icon()
> > +
> > +
> > +
> > +/turf/simulated/floor/fakespace
> > +	icon = 'icons/turf/space.dmi'
> > +	icon_state = "0"
> > +	floor_tile = /obj/item/stack/tile/fakespace
> > +	broken_states = list("damaged")
> > +
> > +/turf/simulated/floor/fakespace/New()
> > +	..()
> > +	icon_state = "[rand(0,25)]"
>
> I thought about doing it that way but it doesn't really make sense for
> carpeting to exactly mirror the underlying space. Maybe if they were were
> glass-bottom stations or something that would be good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/11057/files#r36583451>.
>